packages/docs/src/locales/en/en.json

```diff
@@ -4697,5 +4705,29 @@
       "description": "If you use vue-i18n we can recommend to store VuesticUI messages under specific key",
       "exampleConfig": "Recommended config structure"
     }
+  },
+  "webComponents": {
+    "title": "Web components",
+    "description": "Vuestic UI provides web-components build, so you can pick some component and use them without Vue application",
+    "installation": {
+      "title": "Installation",
+      "description": "Web components build shiped within `vuestic-ui` package and can be imported with suffix `vuestic-ui/web-components`.",
+      "create": "Before you can use web-components they must be registered with `window.customElement`. We provide a nice helper, so you can register specific components:",
+      "installVuestic": "See \"Install Vuestic UI\""
+    },
+    "css": {
+      "title": "CSS",
+      "description": "Since Custom Elements have their own scoped CSS in Shadow DOM, there might be a case when you want to provide some CSS to components. You can do this with `css` property in `registerVuesticWebComponents`. For example, if you want to use Font Awesome in Vuestic components you need provide this CSS:"
+    },
+    "example": {
+      "title": "Usage example",
+      "description": "We use Vuestic web component with vanilla Vite project here",
+      "steps": [
+        "First, we need to register Custom Elements",
+        "Then we can use them in our HTML",
+        "You can also use Global Config with VaConfig element"
+      ],
+      "seeMore": "See more detailed example in our Sandbox"
```

Comment:
Link at page with this anchor leads to 404. Just don't forget about it.

Reply:
For sure. Blindmade URL.